docs/application/native/tutorials/details/native-removed-history.md

```diff
@@ -0,0 +1,39 @@
+| Multimedia - Player | player_set_sound_type() | Mobile & Wearable | since 3.0 | 5.0 | Better alternative | player_set_sound_stream_info() |
+| Network - Bluetooth | bt_adapter_le_start_device_discovery(), bt_adapter_le_stop_device_discovery(), bt_adapter_le_set_device_discovery_state_changed_cb(), bt_adapter_le_unset_device_discovery_state_changed_cb(), bt_adapter_le_add_advertising_data(), bt_adapter_le_remove_advertising_data(), bt_adapter_le_start_advertising(), bt_gatt_foreach_primary_services(), bt_gatt_get_service_uuid(), bt_gatt_foreach_included_services(), bt_gatt_set_characteristic_changed_cb(), bt_gatt_unset_characteristic_changed_cb(), bt_gatt_watch_characteristic_changes(), bt_gatt_unwatch_characteristic_changes(), bt_gatt_get_characteristic_declaration(), bt_gatt_set_characteristic_value(), bt_gatt_read_characteristic_value(), bt_gatt_set_characteristic_value_request(), bt_gatt_clone_attribute_handle(), bt_gatt_destroy_attribute_handle(), bt_gatt_discover_characteristics(), bt_gatt_discover_characteristic_descriptor() | Mobile | since 2.3.1 | 5.0 | Better alternatives | New functions in the Network - Bluetooth | 
+| Social - Service Adaptor | all functions of this module | Mobile & Wearable | since 3.0 | 5.0 | No longer available | - |
+| Telephony - Telephony Information | telephony_call_get_voice_call_state(), telephony_call_get_video_call_state() | Mobile & Wearable | since 3.0 | 5.0 | Better alternatives | telephony_call_get_status() |
+| UI - EFL - Eldbus | all functions of this module | Mobile & Wearable | since 4.0 | 5.0 | Security | - |
+| Multimedia - Recorder | RECORDER_ERROR_SOUND_POLICY, RECORDER_ERROR_SOUND_POLICY_BY_CALL, RECORDER_ERROR_SOUND_POLICY_BY_ALARM of recorder_error_e | Mobile & Wearable | since 3.0 | 5.0 | No longer available | - |
+| Multimedia - Recorder | RECORDER_POLICY_SOUND, RECORDER_POLICY_SOUND_BY_CALL, RECORDER_POLICY_SOUND_BY_ALARM of recorder_policy_e | Mobile & Wearable | since 3.0 | 5.0 | No longer available | - |
+| Multimedia - Camera | CAMERA_ERROR_SOUND_POLICY, CAMERA_ERROR_SOUND_POLICY_BY_CALL, CAMERA_ERROR_SOUND_POLICY_BY_ALARM of camera_error_e | Mobile & Wearable | since 3.0 | 5.0 | No longer available | - |
+| Multimedia - Camera | CAMERA_POLICY_SOUND, CAMERA_POLICY_SOUND_BY_CALL, CAMERA_POLICY_SOUND_BY_ALARM of camera_policy_e | Mobile & Wearable | since 3.0 | 5.0 | No longer available | - |
+| UI - EFL UTIL | efl_util_notification_window_level_error_cb(), efl_util_set_notification_window_level_error_cb(), efl_util_unset_notification_window_level_error_cb(), efl_util_window_screen_mode_error_cb(), efl_util_set_window_screen_mode_error_cb(), efl_util_unset_window_screen_mode_error_cb() | Mobile & Wearable | since 3.0 | 5.0 | No longer available | - |
+
+## Related information
+
+- [Tizen Native API Reference](../../api/)

```

Comment:
During previous review, I mis-typed the path to go. Would you update the link to `(../../api/overview.md)`? Sorry for the inconvenience.